pkg/database/db_test.go

```diff
@@ -75,9 +82,104 @@ func TestCreateQuickstartWithBundle(t *testing.T) {
 		DB.Find(&quickStarts)
 		DB.Model(&tag).Association("Quickstarts").Find(&quickStartsAssociations)
 		assert.Equal(t, dbTag.ID, tag.ID)
-		assert.Equal(t, 1, len(quickStarts))
+		assert.Equal(t, 6, len(quickStarts))
```

Comment:
The number should be probably based on the number of quickstart yaml files. Content folks will be adding new files and we don't want them to touch test files.